### PR TITLE
Show package slugs instead of names

### PR DESCRIPTION
**Pull request by kielabokkie**

### CODE REVIEW

File: src/Skeletor/Packages/Package.php [hunk 1]
```diff
@@ -67,8 +67,12 @@ public function setName(string $name)
     /**
      * @return string
      */
-    public function getVersion()
+    public function getVersion(bool $allowEmpty = true)
     {
+        if ($allowEmpty === false) {
+            return empty($this->version) === true ? 'latest' : $this->version;
```
Comment (titopixelfusion) [inline src/Skeletor/Packages/Package.php:73]: we do not need the === true

File: src/Skeletor/Packages/PackageInterface.php [hunk 2]
```diff
@@ -8,9 +8,9 @@ public function getInstallSlug();
     public function setInstallSlug(string $installSlug);
     public function getName();
     public function setName(string $name);
-    public function getVersion();
+    public function getVersion(bool $allowEmpty);
```
Comment (titopixelfusion) [inline src/Skeletor/Packages/PackageInterface.php]: In Package the signature has a default value of true, don't we need it here too?

**Review by titopixelfusion — APPROVED** [review]
we don't seem to be using PHP-CS-Fixer, we could add it to the required-dev and let it automatically fix the code.

File: src/Skeletor/Packages/PackageInterface.php (hunk 2 above)
Comment (kielabokkie) [reply src/Skeletor/Packages/PackageInterface.php]: Yep true :)

File: src/Skeletor/Packages/Package.php (hunk 1 above)
Comment (kielabokkie) [reply src/Skeletor/Packages/Package.php:73]: I guess this one is personal preference. I like it for readability, but you are right that it is not really required.

**Comment by thijsbouwes** [comment]
Do we want to leave the name parameter of the packages?
Otherwise we can remove all references, like `setName` and stuff.

File: src/Skeletor/Manager/PackageManager.php [hunk 3]
```diff
@@ -119,16 +119,15 @@ public function getAvailablePackageVersions()
     public function specifyPackagesVersions(array $packages)
     {
         $versions = $this->getAvailablePackageVersions();
-        foreach ($packages as $key => $package)
-        {
+        foreach ($packages as $key => $package) {
             $this->cli->br()->yellow(sprintf('Available %s versions: %s', $package->getName(), implode(', ', $versions[$package->getInstallSlug()])));
-            $input = $this->cli->input(sprintf('%s version [%s]:', $package->getName(), $package->getVersion() ));
+            $input = $this->cli->input(sprintf('%s version [%s]:', $package->getName(), $package->getVersion()));
             $versions[$package->getInstallSlug()][] = '';
```
Comment (thijsbouwes) [inline src/Skeletor/Manager/PackageManager.php]: Do we want to show here `getInstallSlug()` ?

Comment (kielabokkie) [reply src/Skeletor/Manager/PackageManager.php]: Good catch, @ThijsBouwes. Updated! :)

**Comment by kielabokkie** [comment]
@ThijsBouwes I think for now we leave it in. Might still need it in the future.

**Review by thijsbouwes — APPROVED** [review]

**Pull request merged**